process/living-cr-rec.html

```diff
@@ -0,0 +1,145 @@
+          solutions, the Working Group should consider moving the document to the <a href='https://www.w3.org/Consortium/Process/#abandon-draft'>Discontinued Draft</a> status.</p>
+
+        <h2 id="cr">When should a Group move to Recommendation?</h2>
+        <p>
+          Unless one or more of the 3 conditions to remain in Candidate Recommendation are valid, a Working Group is strongly
+          encouraged to move forward to Recommendation.
+        </p>
+        <p>
+          Getting to Recommendation is the only way to receive the full endorsement from the W3C for a technical specification.
+          <strong>Candidate Recommendations are not endorsed by W3C.</strong>
+        </p>
+
+        <h2 id="rec">When should a Group keep a document as a Recommendation?</h2>
+
+        <p>
+          A document remains in Recommendation if one or more of the conditions apply:
+        </p>
+        <ol>
+          <li>A meaningful/core subset of the document can be normative and received adequate implementation experience</li>
+          <li>Future corrections and improvements can be dealt through a revision of the Recommendation, including for new features</li>
+        </ol>
+        <p>Note: A Working Group should not move to Proposed Recommendations without considering how to maintain/revise the future Recommendation.</p>
+
+        <h3 id="rec_cons">Ensuring the future of your Recommendation</h3>
+        <p>
+          Here are considerations to allow a Recommendation to be revised:
+        </p>
+        <ul>
+          <li>Editorial updates can be made at any time on a Recommendation, without further review</li>
+          <li>Candidate corrections to make substantive changes can be made at any time on a Recommendation, without further review. Those candidate corrections are not considered normative.</li>
+          <li>Proposed corrections to make substantive changes can be made at any time on a Recommendation, but do trigger reviews.  Those proposed corrections are not considered normative.</li>
+          <li>Similarly, if allowed by the Proposed Recommendation, candidate and proposed additions to add new features can be made at any time on a Recommendation</li>
+          <li>Once proper reviews are done, proposed corrections and proposed additions can be made normative.</li>
+        </ul>
+
+        <h2 id="rec">When should a Group move a Recommendation back to Working Draft or Candidate Recommendation?</h2>
+
+        <p>
+          A document should be moved back to Working Draft or Candidate Recommendation if one or more of the conditions apply:
+        </p>
+        <ol>
+          <li>New features and the </li>
+          <li>Future corrections and improvements can be dealt through a revision of the Recommendation, including for new features</li>
+        </ol>
+        <p>Note: A Working Group should not move to Proposed Recommendations without considering how to maintain/revise the future Recommendation.</p>
```

Comment:
this is copy/pasta, and should be in one place or another. And I don't think it belongs here, so propose deleting
```suggestion
```